src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseApprovalServiceTests.cs

```diff
@@ -991,38 +996,140 @@ public async Task GetReleaseStatuses()
+            {
+                PreviousVersionId = null,
+                ReleaseStatuses = new List<ReleaseStatus>
+                {
+                    new()
+                    {
+                        InternalReleaseNote = "Third",
+                        ApprovalStatus = ReleaseApprovalStatus.Approved,
+                        Created = new DateTime(2000, 10, 10),
+                        CreatedBy = new User {Email = "third@test.com"}
+                    }
+                }
+            };
 
-                Assert.Equal(releaseStatuses[1].InternalReleaseNote, resultStatuses[1].InternalReleaseNote);
-                Assert.Equal(releaseStatuses[1].ApprovalStatus,resultStatuses[1].ApprovalStatus);
-                Assert.Equal(releaseStatuses[1].Created, resultStatuses[1].Created);
-                Assert.Equal(releaseStatuses[1].CreatedBy?.Email, resultStatuses[1].CreatedByEmail);
+            var ignoredRelease2 = new Release
+            {
+                PreviousVersionId = Guid.NewGuid(),
+                ReleaseStatuses = new List<ReleaseStatus>
+                {
+                    new()
+                    {
+                        InternalReleaseNote = "Fourth",
+                        ApprovalStatus = ReleaseApprovalStatus.Draft,
+                        Created = new DateTime(2000, 9, 9),
+                        CreatedBy = new User {Email = "fourth@test.com"}
+                    }
+                }
+            };
 
-                Assert.Equal(releaseStatuses[2].InternalReleaseNote, resultStatuses[2].InternalReleaseNote);
-                Assert.Equal(releaseStatuses[2].ApprovalStatus,resultStatuses[2].ApprovalStatus);
-                Assert.Equal(releaseStatuses[2].Created, resultStatuses[2].Created);
-                Assert.Equal(releaseStatuses[2].CreatedBy?.Email, resultStatuses[2].CreatedByEmail);
+            var contextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = InMemoryApplicationDbContext(contextId))
+            {
+                await contentDbContext.AddRangeAsync(
+                    originalRelease, amendedRelease, ignoredRelease1, ignoredRelease2);
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            await using (var contentDbContext = InMemoryApplicationDbContext(contextId))
+            {
+                var releaseRepository = new ReleaseRepository(contentDbContext,
+                    Mock.Of<StatisticsDbContext>(),
+                    Mock.Of<IMapper>());
+                var releaseService = BuildService(contentDbContext, releaseRepository: releaseRepository);
+                var result = await releaseService.GetReleaseStatuses(amendedRelease.Id);
+
+                var resultStatuses = result.AssertRight();
+                Assert.Equal(2, resultStatuses.Count);
+
+                // originalRelease's ReleaseStatus appears first due the ReleaseStatus's Created date
```

Comment:
Might want most recent first like we show the most recent data first in a lot of other places I think?